### DISCUSSION TITLE
nginx-ui 可不可以内置一些命令

### DISCUSSION THREAD

**Discussion by 1fd** [0xJacky/nginx-ui#710] [General]
nginx-ui 可不可以内置一下命令
 比如安装 ngxin  nginx的重启 重载  更新 或者编译安装 选择编译的模块

**Comment by 0xJacky** [comment]
nginx-ui 在保存配置文件，更新证书后自动reload，右上角也的按钮也有运行状态的指示，可以restart 或者 reload 相关命令的自定义请看 nginxui.com

**Reply by 1fd** [reply]
右上角喔以为是nginx-ui的重启 重载，非docker安装的不是还要手动安装nginx，这个可不可以在面板种实现安装 ngxin  更新 或者编译安装

**Reply by 0xJacky** [reply]
ubuntu的话 直接 apt install nginx 就好了吧
